openstack/blockstorage/v2/extensions/volumeactions/requests.go

```diff
@@ -172,3 +172,35 @@ func TerminateConnection(client *gophercloud.ServiceClient, id string, opts Term
 	})
 	return
 }
+
+// ExtendSizeOptsBuilder allows extensions to add additional parameters to the
+// ExtendSize request.
+type ExtendSizeOptsBuilder interface {
+	ToVolumeExtendSizeMap() (map[string]interface{}, error)
+}
+
+// ExtendSizeOpts contain options for extending the size of an existing Volume. This object is passed
+// to the volumes.ExtendSize function.
+type ExtendSizeOpts struct {
+	NewSize int `json:"new_size,omitempty"`
```

Comment:
What units is `NewSize`? GBs?

Should this be required? If so:
```
NewSize int `json:"new_size" required:"true"`
```

What happens if a value less than the current size is provided?